awx/main/conf.py

```diff
@@ -246,18 +246,6 @@ def _load_default_license_from_file():
     category_slug='jobs',
 )
 
-register(
-    'AWX_ISOLATED_VERBOSITY',
```

Comment:
👍 

This used to be an explicit thing, but when we moved to `ansible-runner` last year and refactored this code, we decided to just make the verbosity match the verbosity defined on the job run itself:

https://github.com/ansible/awx/commit/99478f5d255986cba796b78551a4e2eaf5ef5f17